python/test/unit/runtime/test_interpreter.py

```diff
@@ -0,0 +1,37 @@
+import pytest
```

Comment:
The test should be put in `test_core.py`

Reply:
Ok I'll move it there!